starter-code/package.json

```
@@ -3,7 +3,8 @@
   "version": "2.0.0",
   "main": "app.js",
   "scripts": {
-    "start": "node app.js"
+    "start": "node app.js",
+    "dev": "nodemon app.js"
```

Si van a trabajar con partials este script puede ayudarlos a que el server reinicie viendo los cambios incluso de los partials.

```suggestion
    "dev": "nodemon -e js,hbs,json,css app.js"
```